### PR TITLE
Support parsing leakreports with excluded leaks

### PR DESCRIPTION
**Pull request by berikv**

### CODE REVIEW

**Review by igorsmotto — APPROVED** [review]

File: Sources/Symbolicator/Models/MemoryLeakReport.swift [hunk 1]
```diff
@@ -3,11 +3,13 @@ import Foundation
 struct MemoryLeakReport: Codable {
     let headers: String
     let leaks: [Leak]
+    let excludedLeakCount: Int
     let binaryImages: String?
 
-    init(headers: String, leaks: [Leak], binaryImages: String?) {
+    init(headers: String, leaks: [Leak], excludedLinkCount: Int, binaryImages: String?) {
```
Comment (amanjeetsingh150) [inline Sources/Symbolicator/Models/MemoryLeakReport.swift:9]: Shouldn't this be excludedLeakCount?

**Comment by amanjeetsingh150** [comment]
Can you tell what is the name of the field that is added to JSON?

File: Sources/Symbolicator/Models/MemoryLeakReport.swift (hunk 1 above)
Comment (berikv) [reply Sources/Symbolicator/Models/MemoryLeakReport.swift:9]: Nice find! I'll update the pr

**Comment by berikv** [comment]
> Can you tell what is the name of the field that is added to JSON?

It'll be the same as the field name. So `excludedLeakCount` after the rename.

**Pull request merged**